tests/Unit/ConfigTest.php

```diff
@@ -31,9 +31,13 @@
 });
 
 it('should be able to create a peck.json config file', function (): void {
-    $configFilePath = __DIR__.'/../../peck.json';
-    $backup = $configFilePath.'.backup';
-    rename($configFilePath, $backup);
+    if (file_exists($filePath = dirname(__DIR__).'/peck-new.json')) {
```

Comment:
Maybe declare the variable before? It could improve readability.
Furthermore, is it possible to have this config lie in the `tests/` directory and not at the root level?